cli.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require 'thor'
+require 'exif'
+
+class CLI < Thor
+  desc "scan", "Recursively reads all of the images from the supplied directory"
+  option :path, :type => :string, :required => false, :desc => "Optional parameter that allows any other directory to be passed in."
+
+  def scan
+    report = File.open("report.csv", "w")
```

Comment:
i think the garbage collector will automatically close it, but i think it's usually a good practice to close the file.

another option is to use `File.open` with a block, and it will automatically close it for you at the end of the block (the block receives the opened file, so you can do `file.write("your text") `)

Reply:
good catch. Updated